adapter/test_slack.py

```diff
@@ -184,6 +220,8 @@ async def test_success_integration_ask_chat(self, mock_slack_adapter, mock_reque
 
         await slack_adapter.ask(mock_request)
 
+        time.sleep(1)
```

Comment:
Is the time sleep really necessary to simulate this test case?

Reply:
I haven't found a better way but this way works for now. It's necessary to ensure the child thread used to generate and send a response has resolved properly so the rest of the assertions are correct